src/main/java/teammates/common/datatransfer/CourseRoster.java

```diff
@@ -104,4 +128,42 @@ private void populateInstructorListByEmail(List<InstructorAttributes> instructor
             instructorListByEmail.put(i.email, i);
         }
     }
+
+    /**
+     * Gets the info of a participant associated with the identifier in the course.
+     *
+     * @see CourseRoster#INFO_NAME
+     * @see CourseRoster#INFO_TEAM_NAME
+     * @see CourseRoster#INFO_SECTION_NAME
+     */
+    public String[] getInfoForIdentifier(String identifier) {
+        String name = Const.USER_NOBODY_TEXT;
+        String teamName = Const.USER_NOBODY_TEXT;
+        String sectionName = Const.DEFAULT_SECTION;
+
+        boolean isStudent = this.getStudentForEmail(identifier) != null;
+        boolean isInstructor = this.getInstructorForEmail(identifier) != null;
+        boolean isTeam = this.getTeamToMembersTable().containsKey(identifier);
+        if (isStudent) {
+            StudentAttributes student = this.getStudentForEmail(identifier);
+
+            name = student.getName();
+            teamName = student.getTeam();
+            sectionName = student.getSection();
+        } else if (isInstructor) {
+            InstructorAttributes instructor = this.getInstructorForEmail(identifier);
+
+            name = instructor.getName();
+            teamName = Const.USER_TEAM_FOR_INSTRUCTOR;
+            sectionName = Const.DEFAULT_SECTION;
+        } else if (isTeam) {
+            StudentAttributes teamMember = this.getTeamToMembersTable().get(identifier).iterator().next();
+
+            name = identifier;
+            teamName = identifier;
+            sectionName = teamMember.getSection();
+        }
+
+        return new String[] { name, teamName, sectionName };
```

Comment:
While this works, we can just define another small data structure with 3 fields for this. That way we also don't need the `INFO_*` variables.